src/pages/DBSettingsPage/index.jsx

```diff
@@ -289,7 +291,7 @@ class DBSettingsPage extends React.Component {
     this.setState({
       gettingPassword: true,
     });
-    handleGetRequest(`/projects/${projectID}/databases/${databaseID}/password`)
+    handleGetRequest(`${DATABASE_API_URL}/databases/${databaseID}/password`)
```

Comment:
You could modify `handleGetRequest` implementation to something like 
```
export const handleGetRequest = (endpoint, axiosInstance = axios) => {
  return new Promise((resolve, reject) => {
    axiosInstance
      .get(endpoint)
      .then((response) => {
        resolve(response);
      })
      .catch((error) => {
        reject(error);
      });
  });
};
```
we are making it have an optional argument, for an Axios instance
```suggestion
    handleGetRequest(`${DATABASE_API_URL}/databases/${databaseID}/password`, databaseAxios)
```

Reply:
sounds good 👍🏾